doc/classes/TileMap.xml

```diff
@@ -202,7 +202,8 @@
 			<return type="Vector2" />
 			<argument index="0" name="map_position" type="Vector2i" />
 			<description>
-				Returns the local position corresponding to the given tilemap (grid-based) coordinates.
+				Returns the center of the cell at the given tilemap (grid-based) coordinates.
```

Comment:
We need to keep the "local" precision from the original version. Since the "map_to_world" naming may mislead user into thinking the result is in global coordinates, and not in local ones.
Maybe something like that: `Returns the center of the cell, as a local position, at the given tilemap (grid-based) coordinates.